packages/web/src/pages/chat-page/components/ChatComposer.tsx

```diff
@@ -235,24 +156,21 @@ export const ChatComposer = (props: ChatComposerProps) => {
     }
     if (chatId !== chatIdRef.current) {
       setValue('')
-      setTrackId(null)
-      setCollectionId(null)
+      reset()
```

Comment:
nit: maybe should call this `resetAudiusLinks` or something

Reply:
maybe it should just be called cleanup. it doesn't actually reset. just clears and should be called on submit. maybe could just call it onSubmit

Reply:
clears state*

Reply:
mostly want to make it clear that this came from `useAudiusLinkResolver`. maybe `clearAudiusLinks`? `resetAudiusLinks`?

Reply:
`onSubmit` feels too overloaded with form submission stuff imo.. 🤷  up 2 u

Reply:
cool